MIGRATION.md

```diff
@@ -266,6 +267,10 @@ In the meantime, these migration notes are the best available documentation on t
 
 ### 7.0 breaking changes
 
+#### Dropped support for Node 15 and below
+
+Storybook 7.0 requires Node 16 or above. If you are using olders version of Node, you will need to upgrade, or keep using Storybook 6 in the meantime.
```

Comment:
This is great, let's also update the number below in `#### Modern browser support`

>and node code is transpiled with a target of `node >= 14`